locales/en-US/messages.properties

```diff
@@ -99,7 +99,7 @@ share=Share
 tell_your_friends=Tell your friends!
 help_spread_the_word=We need your help to spread the word.
 # Used in a tweet, please keep below 116 characters
-i_donated_to_mozilla=I donated to @mozilla today because I #lovetheweb. Join me and help build + protect the Web forever
```

Comment:
Please update the key here and in the code, otherwise it won’t be translated.
We also need to update the comment, since Twitter limit has changed. I think we can go with 257 characters now

Reply:
What should the key be changed to? Also, what should the copy be?

Reply:
Copy is correct, as far as I can tell. For key/comment, we could go with the following:

Updated key example: `i_donated_to_mozilla_2017`
Updated comment: `# Used in a tweet, please keep below 257 characters`